### PR TITLE
DSE 5.1.28 Release Notes

### PR DESCRIPTION
**Pull request by thobe**
Adds 5.1.28 release notes.

### CODE REVIEW

File: DSE_5.1_Release_Notes.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+ * Apache Solr™ 6.0.1.0.2882
+ * Apache Spark™ 2.0.2.42
+ * Apache TinkerPop™ 3.2.11-20210716-faea8d16
+ * Apache Tomcat® 8.5.72&ast;
+ * DSE Java Driver 1.8.3-dse+20201217
+ * Netty 4.0.54.1.dse
+ * Spark JobServer 0.6.2.240
+
+## 5.1.28 DSE Cassandra
+* Await timeout for shutting down non periodic tasks is now configurable with the new jvm option {{cassandra.non_periodic_tasks_shutdown_timeout_in_minutes}}. When timeout is reached, force shutdown those tasks. (DSP-22241)
```
Comment (tiagomlalves) [inline DSE_5.1_Release_Notes.md:18]: Here we should replace jira formatting by Markdown formatting (i.e. replace {{ and }} by ` )

**Review by tiagomlalves — APPROVED** [review]

File: DSE_5.1_Release_Notes.md (hunk 1 above)
Comment (thobe) [reply DSE_5.1_Release_Notes.md:18]: Ok. Will do.

**Comment by thobe** [comment]
With the release being a bit delayed, I will need to update the date before merging this.

**Comment by tiagomlalves** [comment]
Good to go! :)

**Pull request merged**